Dockerfile

```diff
@@ -25,35 +27,33 @@ RUN cp -r build/* /app/static
 RUN mv /app/static/static/* /app/static/
 RUN rm -r /app/static/static
 
+# Set up Django
+FROM python:alpine
 
-FROM python:buster
+# Get PostgreSQL client
+# https://stackoverflow.com/a/47871121
+RUN apk update
+RUN apk add --no-cache postgresql-libs
+RUN apk add --no-cache --virtual build-dependencies gcc musl-dev postgresql-dev
 
-# Copy contents from builder
-COPY --from=builder /app/static /app/static
+# Copy static site from webapp
+COPY --from=webapp /app/static /app/static
 
-# Copy base files
-COPY banwebScrape.py /app/
-COPY course_gather /app/course_gather
-COPY manage.py /app/
-COPY requirements.txt /app/
-COPY scraper /app/scraper
+WORKDIR /app
 
-WORKDIR /app/
+COPY requirements.txt .
 
-# Set environment variables
-ARG DJANGO_SETTINGS_MODULE
-ENV DJANGO_SETTINGS_MODULE=$DJANGO_SETTINGS_MODULE
+# Install Python dependencies
+RUN pip3 install --upgrade pip -r requirements.txt --no-cache-dir
+
+# Remove unneeded build dependencies for smaller image
+RUN apk del build-dependencies
 
-ARG SECRET_KEY
-ENV SECRET_KEY=$SECRET_KEY
+# Set environment variables
+ENV DJANGO_SETTINGS_MODULE="course_gather.settings.prod_settings"
 
-# Install python deps and build backend
-RUN pip3 install --upgrade pip -r requirements.txt
-RUN python3 ./manage.py makemigrations
-RUN python3 ./manage.py migrate
-RUN python3 ./banwebScrape.py
-RUN python3 ./manage.py custom_db_sync /app/writer.csv
+# Copy source
+COPY . .
 
-WORKDIR /app/
 EXPOSE 8000
-CMD ["gunicorn", "-c", "course_gather/conf/gunicorn.ini", "course_gather.wsgi:application"]
```

Comment:
This should remain CMD instead of ENTRYPOINT for prod

Reply:
You're right, my bad. Fixed.